package.json

```
@@ -24,11 +24,24 @@
     "atom-linter": "10.0.0",
     "atom-package-deps": "5.1.0",
     "consistent-env": "1.3.1",
-    "uuid4": "1.1.4"
+    "uuid4": "1.1.4",
+    "atom-languageclient": "^0.9.8"
```

Use a pinned version, updates (including the `package-lock.json` that makes ranges useless) will be handled by Renovate.
```suggestion
    "atom-languageclient": "0.9.8"
```